### PR TITLE
Selects Projects

### PR DESCRIPTION
**Pull request by dwagg**

### CODE REVIEW

**Comment by dwagg** [comment]
There are not any unit tests with this branch. I could not find a way to call the functions in the CollectionViewCell class, I think that is handles by the os. The only other real code added was the code to navigate to the next screen. These things may make sense to UI test but there is no complex logic we are not testing and I could not find a way to even hit these methods in a unit test.

File: Mobile Test Runner for Jama/Mobile Test Runner for Jama/ProjectCollectionViewCell.swift [hunk 1]
```diff
@@ -10,4 +10,19 @@ import UIKit
 
 class ProjectCollectionViewCell: UICollectionViewCell {
     @IBOutlet weak var projectCellLabel: UILabel!
+    
+    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
+        super.touchesBegan(touches, with: event)
+        self.backgroundColor = UIColor(colorLiteralRed: 0xF5/0xFF, green: 0xF5/0xFF, blue: 0xF5/0xFF, alpha: 1)
```
Comment (mmcbee1) [inline Mobile Test Runner for Jama/Mobile Test Runner for Jama/ProjectCollectionViewCell.swift:16]: Are the projects supposed to turn red when selected? In the simulator they become a little grey - is that what this line is doing?

**Review by mmcbee1 — APPROVED** [review]
Works as intended according to all acceptance criteria listed on the ticket in Trello. Approved with a comment for clarification, just for my own understanding

File: Mobile Test Runner for Jama/Mobile Test Runner for Jama/ProjectCollectionViewCell.swift (hunk 1 above)
Comment (dwagg) [reply Mobile Test Runner for Jama/Mobile Test Runner for Jama/ProjectCollectionViewCell.swift:16]: Turn red? I wasn't sure if we had anything designed for this is I just freestyles the UI. It is only supposed to turn light grey when pressed. I am not married to it if we find something that looks better.

**Pull request merged**